src/main/resources/db/migration/0350/V367__FunderReports.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE funder.funder_reports (
```

Comment:
Maybe `published_reports` would make the intent of these tables clearer, since that's the terminology we're using in product discussions.